app/src/main/java/net/vonforst/evmap/model/ChargersModel.kt

```diff
@@ -296,9 +299,24 @@ data class Address(
 
 @Parcelize
 @JsonClass(generateAdapter = true)
-data class Chargepoint(val type: String, val power: Double, val count: Int) : Equatable,
-    Parcelable {
-    fun formatPower(): String {
+data class Chargepoint(
+    // The chargepoint type (use one of the constants in the companion object)
+    val type: String,
+    // How many instances of this plug/socket are available?
+    val count: Int,
+    // Power in kW (or null if unknown)
+    val power: Double?,
```

Comment:
Was it intentional to swap the order of `count` and `power` here, which causes a large portion of the changes in this PR? `GEChargepoint` and `OCMChargepoint` still have it the other way round, so it's a bit inconsistent like this...

Intuitively I would also say it makes more sense to have `type` and `power` first, as `count` is the number of plugs of a certain type *and* with a certain power (e.g. there can be multiple `Chargepoints` with the same `type`, but different `power`s).

Reply:
Yes, it was intentional because it feels wrong to have optional fields before required fields. A chargepoint will always have a count (you know it's there, so it's at least 1) but the output power isn't always known.

If you want, I can change it back, or I can update `GEChargepoint` and `OCMChargepoint` as well (I missed those). No strong opinions there 🙂

Reply:
It is an optional field in the sense that it is nullable, but not an optional field in the sense that it has a default value and can thus be left out in the constructor call: We still need to write e.g. `Chargepoint("Type 2", 1, null)`, not `Chargepoint("Type 2", 1)`). For the latter to work we would need to define it as `val power: Double? = null`, and yes, in that case it would make sense to move it to the end of the list.

But I do like that the `power` needs to be explicitly set to `null` if it's not available, as it is a pretty important piece of information for the EV driver that should be provided if at all possible. So I would tend towards leaving the order as it was before and just making the `power` nullable 🙂

Reply:
Alright, I amended my commit!